ffig/filters/capi_filter.py

```diff
@@ -52,7 +52,7 @@ def to_go(t):
         if t.pointee.kind == TypeKind.CHAR_S:
             return 'string'
         if t.pointee.kind == TypeKind.RECORD:
-            return t.pointee.name.replace('const ', '')
+            return impl
```

Comment:
For now, we only support pointers back to the class itself, so this works. Ultimately, we need to be a lot smarter about how we handle these things, so that we can support member functions that accept other types.